planetiler-custommap/src/main/resources/samples/shortbread.yml

```diff
@@ -571,43 +615,59 @@ layers:
     - key: ref_rows
       value: '${ size(feature.tags["ref"].split(";")) }'
       exclude_when: *missing_ref
+    - key: tunnel
+      value: true
+      include_when:
+        tunnel: [ yes, building_passage ]
+        covered: yes
+      else: false
 
 - id: street_polygons
   features:
   - source: osm
     geometry: polygon
-    min_zoom: 14
+    min_zoom:
+      default_value: 14
+      overrides:
+        11:
+          area:aeroway: runway
+        13:
+          area:aeroway: taxiway
     include_when:
-      __all__:
-      - highway: [ pedestrian, service ]
-      - area: yes
+      # TODO this is a really strange difference to street_polygons.
+      # https://github.com/shortbread-tiles/shortbread-docs/issues/80
```

Comment:
interesting... what does the tilemaker implementation do?

Reply:
they don't care about `aeroway` and only consider `area:aeroway`

![image](https://github.com/user-attachments/assets/2b3b5a9f-7235-4739-8f8e-f4d81e501b08)

Reply:
schould I change it as such?

Reply:
Seems like a mistake... I see plenty of runway polygons tagged with `area=yes` `aeroway=runway` (not `area:aeroway=runway` ([for example](https://www.openstreetmap.org/way/997816571)).  Think it would make sense to support both in both layers?

Reply:
Not sure.


@pnorman do you have an opinion how plane tiler should handle this?

Reply:
If there's an question with the shortbread spec, it should be raised there.